jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -17,22 +26,31 @@ pipeline {
 
         stage('Clone Operator Repo') {
             steps {
-                //ToDo Clone the integeratly-operator repo
-                echo "Clone Operator Repo"
+                cleanWs()
+                script {
+                    sh 'git clone https://github.com/laurafitzgerald/integreatly-operator.git'
+                }
             }
         }
 
         stage("Ensure Product Branch") {
             steps {
-                //ToDo Create the product next branch i.e. 3scale-next from the target branch if it doesn't exist
-                echo "Ensure Product Branch"
+                script {
+                    dir("${env.WORKSPACE}/integreatly-operator"){
+                        //This branch can be updated later, just using the pushed one for now
+                        sh 'git checkout image-extract'
+                    } 
+                }
             }
         }
 
         stage('Extract and Process Manifests') {
             steps {
-                //ToDo Extract the manifests from a container image and process them as required
-                echo "Extract and Process Manifests"
+                dir("${env.WORKSPACE}/integreatly-operator"){
```

Comment:
Any reason you need ${env.WORKSPACE}, you should already be in that directory, I'd probably also just do a dir round the whole lot after the checkout tbh, not sure how that works in the declarative pipelines though.